projects/ion/src/lib/divider/divider.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input } from '@angular/core';
+
+type Orientation = 'horizontal' | 'vertical';
+
+type Type = 'solid' | 'dashed' | 'text';
```

Comment:
```suggestion
type DividerType = 'solid' | 'dashed' | 'text';
```
It seems that storybook doesn't like types with same name and it isn't showing correctly this type, changing to Type of buttons. Changing this name solved the problem.